machine-learning/allPlayers.csv

```diff
@@ -0,0 +1,20 @@
+Birthday,Nat,Height,Weight,DraftYear,OverallDraft,Hand,Position,Team,GamesPlayed,Goals,Assists,Points,PIM,Shifts,TimeOnIce,FullName,Salary
```

Comment:
Maybe move it to /data folder as well?

Reply:
This wasn't supposed to be checked in! Thanks!